spec/jobs/email_missed_reservations_job_spec.rb

```diff
@@ -1,34 +1,31 @@
 require 'spec_helper'
 
-# see http://robots.thoughtbot.com/test-rake-tasks-like-a-boss
-describe 'email_missed_reservations' do
-  include_context 'rake'
-
+describe EmailMissedReservationsJob, type: :job do
   before(:each) do
     mock_app_config(admin_email: 'admin@email.com', disable_user_emails: false,
                     send_notifications_for_deleted_missed_reservations: true)
     @missed = FactoryGirl.create(:missed_reservation,
                                  start_date: Time.zone.today - 2.days,
                                  due_date: Time.zone.today - 1.day)
-    @not_missed = FactoryGirl.create(:valid_reservation)
+    FactoryGirl.create(:valid_reservation)
   end
 
   it 'updates flags after email is sent' do
-    expect { subject.invoke }.to \
+    expect { EmailMissedReservationsJob.perform_now }.to \
```

Comment:
You can use `described_class` instead of the actual class name to keep things concise.